src/shuffle.ts

```diff
@@ -0,0 +1,41 @@
+import { purry } from './purry';
+
+/**
+ * Creates an array of shuffled values, using the
```

Comment:
I'd phrase this as "Shuffles the input array" as this doesn't create an array per-se (like you'd expect a "randomArray" function would create one)

Reply:
I wanted to indicate that it does not mutate the original array. "Shuffles the input array" sounds like it is mutating.

Reply:
All remeda functions shouldn't mutate the input, that's a core concept of Remeda.

Reply:
Updated